src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/components/enterprise-alert/index.tsx

```diff
@@ -16,34 +16,50 @@ import { isProductSlug, productSlugsToNames } from 'lib/products'
 import s from './enterprise-alert.module.css'
 
 interface EnterpriseAlertProps {
-	productSlug: ProductSlug
+	productSlug: ProductSlug | 'vault-secrets' | 'vault-radar'
```

Comment:
What do you think about changing this to 
```suggestion
	productSlug: ProductSlug | 'hcp-vault-secrets' | 'hcp-vault-radar'
```